_admin/setup/LDAP-config-AD.md

```diff
@@ -6,32 +6,22 @@ last_updated: tbd
 sidebar: mydoc_sidebar
 permalink: /:collection/:path.html
 ---
-ThoughtSpot enables you to set up integration with LDAP using Active Directory. After successful setup, you can authenticate users against secure LDAP.
+ThoughtSpot enables you to set up integration with Active Directory using LDAP. After successful setup, you can authenticate users against AD, including authentication over SSL.
 
 
 ## Before you begin
-Before you configure LDAP for Active Directory, collect the following information:
-- **URL**: This is needed to connect to Active Directory. For example, `ldap://192.168.2.48:389`
-- **Domain name**: Preferred domain under which users who want to be authenticated against Active Directory reside. When a user logs in with a username, the default domain is added to the username before sending it to the LDAP server. If users reside in multiple domains, you can still designate one of them as the default.
-
-  **_NOTE:_** Users belonging to a non-default domain will have to explicitly qualify their username when they log in, for example: `username@ldap1.thoughtspot.com`
-- **Search base**: LDAP search base that allows ThoughtSpot to find user properties such as email and display name from LDAP.
-- **SSL**: Obtain the SSL certificate from an issuing authority if you want to use SSL.
```

Comment:
This information is useful to have but the wording seems wrong. Some other information to keep in mind are:

1. If AD servers are sitting behind a load balancer then the setup should be such that we procure the SSL certificate to identify ourself to the load balancer and the communication after LB is non-secure. 
2. We don't support scenario where there are multiple AD servers and each of them provides its own SSL cert.